simtools/corsika/corsika_runner.py

```diff
@@ -164,12 +172,14 @@ def _load_corsika_data_directories(self):
         self._corsika_log_dir = corsika_base_dir.joinpath("log")
         self._corsika_log_dir.mkdir(parents=True, exist_ok=True)
 
-    def get_run_script(self, **kwargs):
+    def prepare_run_script(self, use_pfp=True, **kwargs):
         """
         Get the full path of the run script file for a given run number.
 
         Parameters
         ----------
+        use_pfp: bool
+            Whether to use the preprocessor in preparing the CORSIKA input file
         run_number: int
```

Comment:
I dont see these 2 following parameters in the declaration of the function. are they part of kwargs? maybe one should indicate that in case affirmative

Reply:
Yes, they are in kwargs. As far as I remember we don't have to indicate that arguments are in kwargs instead of being named arguments. However, if you feel strongly or find that we have to, please let me know and I change it.

Reply:
what I meant is something similar to the this (visualize.py):
```
def plot_hist_2D(data, **kwargs):
    """
    Produce a two dimensional histogram plot. Any option that can be changed after plotting (e.g.,\
    axes limits, log scale, etc.) should be done using the returned plt instance.

    Parameters
    ----------
    data: numpy structured array
          The columns of the structured array are used as the x-axis and y-axis titles.
    **kwargs:
        * title: set a plot title.
        * Any additional kwargs for plt.hist2d
```
I would propose to do the same here but we don't need to go through the code and fix everywhere